### DISCUSSION TITLE
[2 Decimals] Pontos negativos e racional por trás do Real Digital (e o Tokenizado) usar 2 decimais

### DISCUSSION THREAD

**Discussion by Robsonsjre** [bacen/pilotord-kit-onboarding#13] [Ideas]
Oi pessoal, 

Gostaria de discutir o racional por trás da escolha dos 2 decimais para o Real Digital e o Tokenizado.
No passado, tivemos problemas em algumas auditorias por perda de precisão acumulada após algumas operações de divisão. 

Apesar de ter formas de abstrair isso, por exemplo sempre multiplicar por um fator antes de realizar uma operação de divisão, me parece que seria mais simples se os decimais fossem aumentados.

Aberto para aprofundar na discussão do tema.

**Comment by Joaonorr** [comment]
Acredito que como a moeda seria para realizar serviços financeiros, como o pix, não seja necessário a utilização de mais de dois decimais.

O próprio tipo lida com operações nessa razão sem uma discrepante perda de informação.

**Comment by BrunooMoniz** [comment]
@Robsonsjre também acho válido estudar a viabilidade de aumentar os decimais para aumento da precisão.

Respondendo o @Joaonorr, a informação se torna importante no Piloto já que o foco é mais abrangente que pagamentos instantâneos. Acredito que vamos ter operações financeiras no cálculo de rendimento dos títulos públicos tokenizados (TPFt).

**Comment by aldenio** [comment]
Prezados, 

O Real Digital e o Real Tokenizado são expressões diferentes da nossa moeda, o Real.
O Real Digital possui a mesma natureza jurídica e características das reservas bancárias, incluindo o número de casas decimais.
O Real Tokenizado também possui a mesma natureza jurídica e características do ativo que representa, seja ele um depósito a vista ou uma moeda eletrônica, sendo assim também herda o número de casas decimais desse ativo.

Vale salientar que este projeto é apenas um piloto e para os casos de uso elencados para os testes, não há a previsão de perda de precisão pelas operações envolvidas.
